src/src/transport/usb-cdc-uart.c

```diff
@@ -366,7 +366,14 @@ otError otPlatUartSend(const uint8_t *aBuf, uint16_t aBufLength)
 
 otError otPlatUartFlush(void)
 {
-    return OT_ERROR_NOT_IMPLEMENTED;
+    while (sUsbState.mTransferDone)
```

Comment:
Shouldn't it be negated? We also need to ensure proper behavior when otPlatUartSend(). We can go with sUsbState.mTransferInProgress && !sUsbState.mTransferDone (we cannot check sUsbState.mTxBuffer because it is cleared when transmission starts.

Reply:
fixed. I wonder why was this working..